### PR TITLE
Wildfire

### PR DESCRIPTION
**Pull request by Kitri**
Updated routes for wildfire and als bells from these links:
https://www.thecrag.com/climbing/south-africa/waterval-boven/area/1539237549
https://www.thecrag.com/climbing/south-africa/waterval-boven/area/242639592

Note: new routes are grouped by crag id, not id - so ids are no longer chronological. I'm not sure if you generate the json from csv or something?

Also fixed the stars not pulling through to the route info, and added the number of draws to the route-card and the route.

### CODE REVIEW

File: src/components/route-card/route-card.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <ion-card>
   <ion-card-header>
     {{ route.name }}
+    <b *ngIf="route?.number_draws > 0" style="float: right">[{{route?.number_draws}}D]</b>
```
Comment (kylemeenehan) [inline src/components/route-card/route-card.html]: Let's avoid inline styles, rather put into the style sheets and use flex. If you're not comfortable with css, let me know and I'll help out here.

File: src/pages/route/route.html [hunk 2]
```diff
@@ -15,6 +15,7 @@ <h1>{{ route?.name }}</h1>
     <ion-card-header text-right>
         <ion-icon *ngFor="let star of starArray" name="star"></ion-icon>
         {{route?.grade}}
+        &nbsp;<b *ngIf="route?.number_draws > 0" >[{{route?.number_draws}}D]</b>
```
Comment (kylemeenehan) [inline src/pages/route/route.html]: Are you wanting to break this out into a new link? If so, let's rather contain them in divs. Not a fan of &nbsp. Let me know if you need some help here.

File: src/pages/route/route.ts [hunk 3]
```diff
@@ -22,7 +22,7 @@ export class RoutePage {
 
     this.route = this.apiService.getRouteById(this.routeId);
     this.starArray = [];
-    for (let i = 0; i > this.route.starts; i++) {
+    for (let i = 0; i < this.route.stars; i++) {
```
Comment (kylemeenehan) [inline src/pages/route/route.ts]: Haha, thanks for picking this up!

File: src/pages/route/route.html (hunk 2 above)
Comment (Kitri) [reply src/pages/route/route.html]: Nah, just wanted to put a space for styling purposes. I could place it in an <ion-col width-10> which has the same effect, if that works better for you?

**Comment by Kitri** [comment]
I've made changes as you suggested. I'm not sure if the ion-col is the best solution, but it's better than &nbsp :) I'm not extremely familiar with ionic, so not sure if there are better elements to use that would allow a single space left of the text? (Purely for styling purposes, it looks cramped without the space)

File: src/pages/route/route.html (hunk 2 above)
Comment (kylemeenehan) [reply src/pages/route/route.html]: Can you just put a  in and I'll do the styling?

Comment (Kitri) [reply src/pages/route/route.html]: Done :-)

**Pull request merged**